content/blog/2023/03/camel-k-telemetry/index.md

```diff
@@ -0,0 +1,420 @@
+---
+title: "Camel K Observability: Distributed Tracing"
+date: 2023-03-25
+draft: false
+authors: [gansheer]
+categories: ["Camel K", "Features"]
+preview: "Camel K: how to configure Distributed Tracing for an Integration"
+---
+
+Tracing is an important approach for controlling and monitoring the experience of users, it allows us to gather more information about an integration's performance. 
+
+Camel K has long been providing support for distributed tracing using OpenTracing. At the beginning of 2022, the [CNCF](https://www.cncf.io) announced that there were [archiving the OpenTracing project](https://www.cncf.io/blog/2022/01/31/cncf-archives-the-opentracing-project/) in favor of the [OpenTelemetry project](https://opentelemetry.io/). OpenTelemetry is the latest solution created by merging OpenTracing and OpenCensus. As a result, we decided in Camel K 1.12 to introduce the [`telemetry` trait](https://camel.apache.org/camel-k/1.12.x/traits/telemetry.html) based on OpenTelemtry. At the same time we decides to deprecate the [`tracing` trait](https://camel.apache.org/camel-k/1.12.x/traits/tracing.html) based on OpenTracing.
```

Comment:
"Camel K has been providing support for distributed tracing using OpenTracing since long time" sounds better. Also there is some typo in "OpenTelemtry"